### PR TITLE
BCSD Voorzitter Alert

### PR DESCRIPTION
**Pull request by ruben-bottu**
## Description

A warning is shown if the voorzitter van BCSD is not a member of RMW or Vast Bureau

## How to test

- Add a voorzitter to BCSD
- A closeable warning should be shown
- Add that person as a member (lid) to RMW or Vast Bureau and the warning should disappear without needing a refresh

### CODE REVIEW

File: app/components/verkiezingen/bcsd-voorzitter-alert.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    return this.args.bestuursorganenInTijd;
+  }
+
+  // Polling because otherwise changes in the installatievergadering data are not picked up
+  handleErrorMessage = task({ restartable: true }, async () => {
+    if (this.isDestroyed) {
+      return;
+    }
+
+    const voorzitter = await this.getVoorzitterBCSD();
```
Comment (Rahien) [inline app/components/verkiezingen/bcsd-voorzitter-alert.js:32]: somehow the error message isn't shown if you add a new voorzitter that isn't in the required organs until you refresh.

**Review by Rahien — CHANGES_REQUESTED** [review]

File: app/templates/verkiezingen/installatievergadering.hbs [hunk 2]
```diff
@@ -63,7 +63,13 @@
         @bestuursorganen={{@model.bestuursorganenInTijd}}
         @sort={{this.sort}}
         @form={{@model.mandatarisNewForm}}
-      />
+      >{{#if (await bestuursorgaan.isBCSD)}}
+          <Verkiezingen::BcsdVoorzitterAlert
+            @bcsdBestuursorgaanInTijd={{bestuursorgaan}}
+            @bestuursorganenInTijd={{@model.bestuursorganenInTijd}}
+          />
+        {{/if}}
```
Comment (ruben-bottu) [inline app/templates/verkiezingen/installatievergadering.hbs:71]: Is it still useful to have this if in the `yield`? Can't it be moved to `PrepareLegislatuurSection` component?

Comment (JonasVanHoof) [reply app/templates/verkiezingen/installatievergadering.hbs:71]: Why are we not putting this component in the `PrepareLegislatuurSection` component? Everytime an update is done in the table it reloads and the alert check can be triggered. If we do this we get rid of the polling and it is checked on every update. 

Im I missing somehting do this?

**Review by bfidlers — APPROVED** [review]
Moved the warning, but looks good otherwise.

**Pull request merged**